src/framework/ui/view/mainwindowbridge.cpp

```diff
@@ -152,6 +153,17 @@ void MainWindowBridge::toggleFullScreen()
     }
 }
 
+void MainWindowBridge::updateFullScreen()
+{
+    bool isFullScreen = m_window ? m_window->windowStates() & Qt::WindowFullScreen : false;
```

Comment:
This one is causing an MSVC compiler warning:
```
Warning	C4996	'QFlags<Qt::WindowState>::QFlags': Use default constructor instead (compiling source file ...\MuseScore\msvc.build_x64\src\framework\ui\CMakeFiles\ui.dir\Unity\unity_2_cxx.cxx)	ui	...\MuseScore\src\framework\ui\view\mainwindowbridge.cpp	158	
```
which I can't make any sense of?